src/components/ProjectIndex.vue

```diff
@@ -33,13 +37,41 @@
 import json from "../../src/repos.json";
 import RepoChips from "@/components/project/RepoChips.vue";
 import RepoDescription from "@/components/project/RepoDescription.vue";
+import { ref, onMounted } from "vue";
 
 export default {
   components: {RepoChips, RepoDescription},
-  data() {
+  async setup() {
+    const repos = ref([])
+
+    const fetchRepoData = async () => {
+      const repoDataPromises = Object.keys(json).map(async (project) => {
+        const projectInfo = json[project];
+        const mavenPath = projectInfo.artifact.replace(/\./g, '/').replace(':', '/');
+
+        const versions = await fetch(`https://maven.neoforged.net/api/maven/versions/releases/${mavenPath}`)
```

Comment:
For the latest version the path `/api/maven/latest/version/{repository}/{gav}` can be used to save bandwidth.

Reply:
> For the latest version the path `/api/maven/latest/version/{repository}/{gav}` can be used to save bandwidth.

do you mean `/api/maven/latest/version/releases/{artifact}` instead?

Reply:
never mind, it's done